src/types/event-log.ts

```diff
@@ -23,21 +19,35 @@ export interface EventLog {
    * adds an event to a tenant's event log
    * @param tenant - the tenant's DID
    * @param messageCid - the CID of the message
-   * @returns {Promise<string>} watermark
+   * @param indexes - (key-value pairs) to be included as part of indexing this event.
+   */
+  append(tenant: string, messageCid: string, indexes: { [key:string]: string|boolean }): Promise<void>
+
+  /**
+   * Retrieves all of a tenant's events that occurred after the cursor provided.
+   * If no cursor is provided, all events for a given tenant will be returned.
+   *
+   * The cursor is a messageCid.
```

Comment:
I really want to get rid of this method completely in favor of Query, I can do that as a part of this PR or I think it would be better suited as a part of a separate PR to move `EventsQuery` to the `Messages` interface.

Reply:
Supportive, let's then just rename `gt` to `cursor` to get documentation consistency going for now.